packages/server/.env.example

```diff
@@ -43,3 +43,6 @@ DATA_DIR="./data"
 
 # S3 bucket for ARPA Audit reports
 AUDIT_REPORT_BUCKET=arpa-audit-reports
+
+# SQS queue for grants-ingest events
+GRANTS_INGEST_EVENTS_QUEUE_URL="http://localhost:4566/000000000000/grant-ingest-events"
```

Comment:
curious: why all the zeroes in the path? Is that just how localstack does it? `000000000000`

Reply:
Yep, this is a LocalStack convention.